content/en/docs/contribute/participate/_index.md

```diff
@@ -17,7 +17,7 @@ the documentation for Kubernetes as a whole. See
 for more information about the SIG.
 
 SIG Docs welcomes content and reviews from all contributors. Anyone can open a
-pull request (PR), and anyone is welcome to file issues about content or comment
```

Comment:
The existing word “anyone” is idiomatic English (so is “everyone”).

@AvineshTripathi I'm not yet convinced we should make this change - can you explain why it's a good idea?